docs/files/tutorial-get.py

```diff
@@ -20,8 +20,8 @@
 # Filter by using several fields:
 res.data.filter(
   status='OK',
-  created__gt='2018-10-01',
-  created__lt='2025-11-01',
+  created__gt='2018-10-01 01:23:45',
```

Comment:
Why don't you use `datetime.datetime(2018, 10, 1)`? It looks more natural and easier to remember.

Reply:
Done.